handlers/friday.go

```diff
@@ -29,14 +77,66 @@ func FridayHandler(s *discordgo.Session, m *discordgo.MessageCreate) {
 	switch time.Now().Weekday() {
 		case time.Friday:
 			message.Content = "Sextouu família"
-			message.Files = append(message.Files, processGifUrl(fridayGifUrl))
+
+			randomFridayGif := getRandomGif(fridayTrigger)
+			randomFridayGifUrl := extractGifFromGTenor(randomFridayGif, fridayGifUrl)
+
+			message.Files = append(message.Files, processGifUrl(randomFridayGifUrl))
+		case time.Thursday:
+			message.Content = "Quasee, mas ainda não"
+
+			randomThursdayGif := getRandomGif("falta-um-dia")
+			randomThursdayGifUrl := extractGifFromGTenor(randomThursdayGif, fallbackGifUrl)
+
+			message.Files = append(message.Files, processGifUrl(randomThursdayGifUrl))
 		default:
 			message.Content = fmt.Sprintf("Calma família ainda não é sexta! Falta %d dia(s)", daysRemainingToFriday())
+			
+			randomGif := getRandomGif(time.Now().Weekday().String())
+			randomGifUrl := extractGifFromGTenor(randomGif, fallbackGifUrl)
+
+			message.Files = append(message.Files, processGifUrl(randomGifUrl))
 	}
 
 	s.ChannelMessageSendComplex(m.ChannelID, message)
 }
 
+func getRandomGif(q string) (result GTenorMinimalReturn) {
+	req, err := http.NewRequest("GET", gTenorUrl+"/random", nil)
+	if err != nil {
+		fmt.Println("Cannot make a new http Request", err)
+	}
+
+	query := req.URL.Query()
+	query.Add("q", q)
+	query.Add("key", "LIVDSRZULELA")
```

Comment:
Can you change this line to get the token from the environment?

Reply:
I didn't understand, you say to edit specifically the commit that entered this key? Because in the last commit i changed to get from the env